client/fed_linux_test.go

```diff
@@ -918,8 +918,192 @@ func TestObjectPutNonRecursiveDirPath(t *testing.T) {
 		if err := te.Shutdown(); err != nil {
 			log.Errorln("Failure when shutting down transfer engine:", err)
 		}
-		// Throw in a config.Reset for good measure. Keeps our env squeaky clean!
-		server_utils.ResetTestState()
```

Comment:
Discussed offline:

Adding `defer server_utils.ResetTestState()` in the test is unnecessary because the `func NewFedTest(t *testing.T, originConfig string)` in `fed_test_utils/fed.go` already includes `server_utils.ResetTestState()` in the test cleanup function.